typescript/src/schema/proto.ts

```diff
@@ -316,6 +317,24 @@ function pagingResponseType(
   return '.google.protobuf.FieldDescriptorProto.Type.' + type;
 }
 
+// Ignore non-diregapic pagation method where its response type contains a map.
+function ignorePagingMethod(
```

Comment:
Let's rename it to `ignoreMapPagingMethod` here and everywhere, otherwise the name is confusing.

Reply:
agree